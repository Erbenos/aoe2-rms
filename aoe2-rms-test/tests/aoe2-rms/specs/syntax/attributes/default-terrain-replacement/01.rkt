#lang aoe2-rms
; Replace all connecting terrain with road, but replace water with shallows instead.

<LAND-GENERATION>
(base-terrain 'WATER)
(create-player-lands
 (land-percent 100)
 (other-zone-avoidance-distance 10))

<CONNECTION-GENERATION>
(create-connect-all-players-land
 (default-terrain-replacement 'ROAD)
 (replace-terrain 'WATER 'SHALLOW))
