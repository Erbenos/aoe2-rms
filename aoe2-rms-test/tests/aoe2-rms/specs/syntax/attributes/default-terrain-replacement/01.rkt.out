<LAND_GENERATION>
base_terrain WATER
create_player_lands {
  land_percent 100
  other_zone_avoidance_distance 10
}
<CONNECTION_GENERATION>
create_connect_all_players_land {
  default_terrain_replacement ROAD
  replace_terrain WATER SHALLOW
}
